doc/whats_new/v1.2.rst

```diff
@@ -71,6 +71,9 @@ Changelog
   negative likelihood ratios derived from the confusion matrix
   of a binary classification problem. :pr:`22518` by
   :user:`Arturo Amor <ArturoAmorQ>`.
+- |Fix| Allows `csr_matrix` as input for parameter: `y_true` of the :func:`label_ranking_average_precision_score` metric.
+  Also added default parameter of `accept_sparse="csr"` to :func:`check_array()`, which is used to calcuate the `y_true` value.
```

Comment:
This is a user facing change log. We do not need to state how the bug was fixed.

```suggestion

- |Fix| Allows `csr_matrix` as input for parameter: `y_true` of
   the :func:`metrics.label_ranking_average_precision_score` metric.
   :pr:`23442` by :user:`Sean Atukorala <ShehanAT>`
```

(Also there is a character limit in RST files that we try to use)